grails-app/assets/javascripts/streama/controllers/admin-fileManager-ctrl.js

```diff
@@ -35,6 +37,36 @@ angular.module('streama').controller('adminFileManagerCtrl', ['$scope', 'apiServ
 		})
 	};
 
+	$scope.removeMultipleFiles = function() {
+	  if($scope.deletionBulk.length > 0) {
+      var confirmText = "This will delete all selected Files. Do you want to proceed?";
+      alertify.set({ buttonReverse: true, labels: {ok: "Yes", cancel : "Cancel"}});
+      alertify.confirm(confirmText, function (confirmed) {
+        if(confirmed){
+          apiService.video.removeMultipleFilesFromDisk($scope.deletionBulk).success(function () {
+            $scope.deletionBulk.forEach(id => {
```

Comment:
oh, dont use Ecmascript 6! that is not suppoted by a lot of browsers yet. Please use lodash's forEach instead